src/nvim/ui_bridge.c

```diff
@@ -144,6 +145,29 @@ static void ui_bridge_highlight_set_event(void **argv)
   xfree(argv[1]);
 }
 
+static void ui_bridge_option_set(UI *ui, String name, Object value)
+{
+  // Assumes bridge is only used by TUI
```

Comment:
VimR uses the bridge

Reply:
@justinmk @bfredl VimR actually uses the bridge. I only recently started to use the MsgPack API for some non-UI related tasks like getting the list of buffers. Probably VimR should also use the UI MsgPack API, but since I do not plan to implement externalization of widgets anytime soon, it has very low priority... I'll try to use/implement `option_set` and come back here if I run into problems. 🙂

Reply:
@qvacua I would say feel free to continue using the bridge. It's an option that we want to support. It should be very easy to support externalized widgets for bridge UIs, also, if it doesn't already ...